reqs/management/commands/import_reqs.py

```diff
@@ -45,6 +46,33 @@ def policy_from_row(row):
     )
 
 
+class KeywordProcessor:
+    """Creates or retrieves Keyword models"""
+    def __init__(self):
+        self.cache = {}
+
+    @staticmethod
+    def keywords(row):
+        to_return = []
+        for field, value in row.items():
+            if field == 'Other (Keywords)':
+                to_return.extend(kw.strip()
```

Comment:
Can we break this out? Nested for loops in comprehensions are always a little tough to grasp.